### PR TITLE
Adds human death sound

### PR DESCRIPTION
**Pull request by Cyberboss**
:cl:
soundadd: Humanlike mobs now have an audio cue when they die
/:cl:

### CODE REVIEW

File: code/modules/mob/living/carbon/human/death.dm [hunk 1]
```diff
@@ -23,7 +23,7 @@
 	var/obj/item/organ/heart/H = getorganslot(ORGAN_SLOT_HEART)
 	if(H)
 		H.beat = BEAT_NONE
-
+	playsound(src, 'sound/voice/human/death.ogg', 50, TRUE)
```
Comment (GuyonBroadway) [inline code/modules/mob/living/carbon/human/death.dm]: make this a variable intrinsic to the mob.

**Comment by TribeOfBeavers** [comment]
Does this play if you deathgasp?

**Comment by Cyberboss** [comment]
Yes

**Comment by Durkel** [comment]
I get it, I aint laughin, but I get it.

**Comment by MMMiracles** [comment]
oof

**Comment by epochayur** [comment]
it's a slippery slope to *scream and *fart

**Comment by deathride58** [comment]
,,, Is this another oof meme?

**Review by ShizCalev — CHANGES_REQUESTED** [review]
https://dl.dropbox.com/s/3womui08pwy4rmc/death.ogg

cleaned up ending fade

**Comment by nicbn** [comment]
Not the bad meme wave again

**Comment by Jordie0608** [comment]
This better not be the roblox sound.

**Comment by Jordie0608** [comment]
It was.

**Comment by cebutris** [comment]
We had this on Citadel for a while and it made the game 1000% more immersive I don't know why anyone would close this

**Pull request closed without merging**